### ISSUE TITLE
Every model I train with this doesn't do anything. Was anyone able to actually get a working model out of it?

### ISSUE THREAD

**Issue by derto42** [Thund3rPat/kohya_ss-linux#10]

**Comment by HustLion** [comment]
My loss rate becomes NaN and the final lora is not usable. 
![image](https://user-images.githubusercontent.com/12825646/220545143-e1d58956-462a-4fad-8a68-0bd14c09a654.png)

**Comment by Thund3rPat** [comment]
Hi @derto42 , these are the settings I had success with:

```json
{
  "pretrained_model_name_or_path": "",
  "v2": false,
  "v_parameterization": false,
  "logging_dir": "",
  "train_data_dir": "",
  "reg_data_dir": "",
  "output_dir": "",
  "max_resolution": "512,512",
  "learning_rate": "0.0001",
  "lr_scheduler": "constant",
  "lr_warmup": "0",
  "train_batch_size": 1,
  "epoch": "1",
  "save_every_n_epochs": "1",
  "mixed_precision": "fp16",
  "save_precision": "fp16",
  "seed": "1234",
  "num_cpu_threads_per_process": 2,
  "cache_latents": false,
  "caption_extension": ".txt",
  "enable_bucket": true,
  "gradient_checkpointing": false,
  "full_fp16": false,
  "no_token_padding": false,
  "stop_text_encoder_training": 0,
  "use_8bit_adam": true,
  "xformers": true,
  "save_model_as": "safetensors",
  "shuffle_caption": false,
  "save_state": false,
  "resume": "",
  "prior_loss_weight": 1.0,
  "text_encoder_lr": "5e-5",
  "unet_lr": "0.0001",
  "network_dim": 4,
  "lora_network_weights": "",
  "color_aug": false,
  "flip_aug": false,
  "clip_skip": 2,
  "gradient_accumulation_steps": 1.0,
  "mem_eff_attn": true,
  "output_name": "noshirov3",
  "model_list": "custom",
  "max_token_length": "75",
  "max_train_epochs": "",
  "max_data_loader_n_workers": "1",
  "network_alpha": 4,
  "training_comment": "",
  "keep_tokens": "0",
  "lr_scheduler_num_cycles": "",
  "lr_scheduler_power": "",
  "persistent_data_loader_workers": false,
  "bucket_no_upscale": true,
  "random_crop": false,
  "bucket_reso_steps": 64.0,
  "caption_dropout_every_n_epochs": 0.0,
  "caption_dropout_rate": 0,
  "optimizer": "AdamW"
}
```
